pytest-embedded-serial/pytest_embedded_serial/serial.py

```diff
@@ -54,10 +55,24 @@ def __init__(
         self.port = port
         self.baud = baud
 
-        if port is None:
-            raise ValueError('Please specify port')
+        port_location = kwargs.get('port_location', None)
```

Comment:
Here the `port_location` from param or cli is not actually been passed. Should be passed here: https://github.com/espressif/pytest-embedded/blob/main/pytest-embedded/pytest_embedded/plugin.py#L859

You may refer to other cli options/fixtures as examples